src/pages/index.js

```diff
@@ -1,8 +1,8 @@
 import React from "react";
 import {ThemeProvider} from "styled-components";
 
+/*eslint linebreak-style: ["error", "windows"]*/
```

Comment:
Will you remove changes to each of the files with this line spacing and comments?

Reply:
Okay..doing it right now

Reply:
Hey @leecalcote I've removed all the unnecessary changes I've made. Is it okay now? Please let me know if I need to do anything else or if I've missed something.

Reply:
I can still see more than 1000 lines changed.

Reply:
Should I make a fresh new PR?